Modules/Loadable/Markups/Logic/vtkSlicerMarkupsLogic.cxx

```diff
@@ -338,9 +338,11 @@ void vtkSlicerMarkupsLogic::SetActiveListID(vtkMRMLMarkupsNode *markupsNode)
     if (activePlaceNodeClassName && strcmp(activePlaceNodeClassName, "vtkMRMLMarkupsFiducialNode") == 0)
       {
       selectionNode->SetActivePlaceNodeID(NULL);
-      vtkMRMLInteractionNode *interactionNode = vtkMRMLInteractionNode::SafeDownCast(this->GetMRMLScene()->GetNodeByID("vtkMRMLInteractionNodeSingleton"));
-      if (interactionNode && interactionNode->GetCurrentInteractionMode() == vtkMRMLInteractionNode::Place)
+      vtkSmartPointer<vtkCollection> interactionNodes = vtkSmartPointer<vtkCollection>::Take
```

Comment:
Here I reset the interaction mode for all interaction nodes.

Is it the right approach ?

Reply:
I'd say yes